### PR TITLE
Updates for Buildroot 2020.08 and GCC 9.2

### PR DESCRIPTION
**Pull request by jforissier**
Depends on https://github.com/OP-TEE/manifest/pull/171 (and vice versa).

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: br-ext/configs/toolchain-aarch32 [hunk 1]
```diff
@@ -2,8 +2,8 @@ BR2_TOOLCHAIN_EXTERNAL=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
 BR2_TOOLCHAIN_EXTERNAL_PATH="%TOP_DIR%/toolchains/aarch32"
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-linux-gnueabihf"
-BR2_TOOLCHAIN_EXTERNAL_GCC_8=y
-BR2_TOOLCHAIN_EXTERNAL_HEADERS_4_19=y
+BR2_TOOLCHAIN_EXTERNAL_GCC_9=y
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_4_20=y
```
Comment (etienne-lms) [inline br-ext/configs/toolchain-aarch32:6]: Better move to BR2_TOOLCHAIN_EXTERNAL_HEADERS_5_5=y since supported in Buildroot 2020.08.

**Comment by jforissier** [comment]
`Tested-by: Jerome Forissier <jerome@forissier.org> (QEMU, QEMUv8, HiKey, HiKey960)`

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`
`Tested-by: Etienne Carriere <etienne.carriere@linaro.org> (stm32mp1)`

File: br-ext/configs/toolchain-aarch32 (hunk 1 above)
Comment (jforissier) [reply br-ext/configs/toolchain-aarch32:6]:
```
>>> toolchain-external-custom  Configuring
Incorrect selection of kernel headers: expected 5.5.x, got 4.20.x
make[2]: *** [package/pkg-generic.mk:240: /home/jerome/work/optee_repo_qemu/out-br/build/toolchain-external-custom/.stamp_configured] Error 1
```

IIRC, `BR2_TOOLCHAIN_EXTERNAL_HEADERS_x_y` has to do with the version of the kernel headers that the Glibc was build against, I mean the Glibc inside the cross compiler folder. Here with GCC 9.2 downloaded from the Arm web site, it is 4.20 (as shown below) so I think we have no choice here.
```
$ grep LINUX_VERSION_CODE toolchains/aarch32/arm-none-linux-gnueabihf/libc/usr/include/linux/version.h
#define LINUX_VERSION_CODE 267277
```
267277 = 0x04140D in other words v4.20.13.

Comment (etienne-lms) [reply br-ext/configs/toolchain-aarch32:6]: Indeed, sorry, i mixed-up the config switches.

**Pull request merged**